addons/dogtags/stringtable.xml

```diff
@@ -81,6 +81,7 @@
         </Key>
         <Key ID="STR_ACE_Dogtags_IGUI_Description">
             <English>Onscreen display for checking dogtags</English>
+            <German> </German>
```

Comment:
I have no clue what this means...

Reply:
On the player's UI there is a little display for showing the dog tag information.
It's basically a picture of a dogtag with some text overlayed.
It is similar to the vanilla `hint` display

Reply:
"Zeige Informationsfenster wenn Dogtags überprüft werden" oder sowas in der richtung würde ich es nennen

Reply:
This is not a action that is done. like "Show onscreen display" it's a description of what that dialog is.
"Informationsfenster für das Prüfen von dogtags"
maybe "überprüfen" or
"Informationsfenster das angezeigt wird wenn man dogtags überprüft" :D If you wanna really go crazy.
Or maybe just "Informationsanzeige für Dogtags"?

Reply:
It should be `Hundemarke` as that is what the other Translations uses. An option would be `Erkennungsmarke` which is the correct BW term.

Reply:
+1 für `Erkennungsmarke`